service/history/events/cache.go

```diff
@@ -132,7 +132,7 @@ func (e *CacheImpl) validateKey(key EventKey) bool {
 }
 
 func (e *CacheImpl) GetEvent(ctx context.Context, shardID int32, key EventKey, firstEventID int64, branchToken []byte) (*historypb.HistoryEvent, error) {
-	handler := e.metricsHandler.WithTags(metrics.OperationTag(metrics.EventsCacheGetEventScope))
+	handler := e.metricsHandler.WithTags(metrics.OperationTag(metrics.EventsCacheGetEventScope), metrics.NamespaceTag(key.NamespaceID.String()))
```

Comment:
Let's define a new tag for namespaceID then?

Reply:
Done